### PR TITLE
neutron: add option to not verify the SSL certificates

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

**Comment by lebauce** [comment]
run devstack-tests

File: config/config.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ func init() {
 	cfg.SetDefault("agent.topology.netlink.metrics_update", 30)
 	cfg.SetDefault("agent.topology.neutron.domain_name", "Default")
 	cfg.SetDefault("agent.topology.neutron.endpoint_type", "public")
+	cfg.SetDefault("agent.topology.neutron.ssl_skip_verify", false)
```
Comment (nplanel) [inline config/config.go]: ssl_insecure would fit better, like curl and skydive tls options

File: topology/probes/neutron/neutron.go [hunk 2]
```diff
@@ -397,14 +399,28 @@ func (mapper *NeutronProbe) OnNodeDeleted(n *graph.Node) {
...
 
-			client, err := openstack.NewNetworkV2(provider, gophercloud.EndpointOpts{
+			client.HTTPClient = http.Client{
+				Transport: &http.Transport{
+					TLSClientConfig: &tls.Config{
+						InsecureSkipVerify: config.GetBool("agent.topology.neutron.ssl_skip_verify"),
+					},
+				},
+			}
+
```
Comment (nplanel) [inline topology/probes/neutron/neutron.go]: log a Warning() to advice when neutron client is connected in insecure (like our https server)

**Comment by lebauce** [comment]
run skydive-unit-tests

File: config/config.go (hunk 1 above)
Comment (lebauce) [reply config/config.go]: Changed

File: topology/probes/neutron/neutron.go (hunk 2 above)
Comment (lebauce) [reply topology/probes/neutron/neutron.go]: Added warning

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by safchain** [comment]
run devstack-tests

**Pull request merged**